### DISCUSSION TITLE
No PRs when stabilityDays > dependency update frequency?

### DISCUSSION THREAD

**Discussion by timmkrause** [renovatebot/renovate#11380] [Request Help] [answered]
We're using

```
    "stabilityDays": 7,
    "prCreation": "not-pending",
```

and I am observing the following behavior:

![image](https://user-images.githubusercontent.com/2163179/130416431-a670fdba-a581-4ad9-9b2a-811c6afd830e.png)

Is this by design and intentional behavior?

I think I am misunderstanding something, but this would mean if the update frequency of a package is let's say every 3 days and I do have `stabilityDays` of >3 I won't ever receive an update, right?

**Comment by viceice** [comment, marked as answer]
Yes, but you can set `internalChecksFilter=strict` so unstable updates are fully suppressed and you get latest update pr which passes `stabilityDays`

https://docs.renovatebot.com/configuration-options/#internalchecksfilter

**Reply by timmkrause** [reply]
Thanks for the quick answer. It's working as expected.

Wondering now though if the default behavior is fine as it is. One could argue that if there are one or more versions being released with a high(er) frequency, that these are potential bugfixes/hotfixes of the not yet "stabilized" version.

**Reply by viceice** [reply]
You can combine stability days with a update type filter to set `stabilityDays=0` for patch releases. 🤷‍♂️

**Reply by rarkins** [reply]
Let's say a package releases daily. Is it possible to programmatically determine when "stability" is reached? Maybe they're all stable.. maybe none are. One thing is for sure - you can't just measure it in "days".

**Reply by timmkrause** [reply]
That's clear. We introduced it because we do not want to integrate 0-day bugs and live on the cutting edge. If something still breaks, there is at least a chance that it has already been discovered and tracked/discussed.
